### DISCUSSION TITLE
crazyflie radio

### DISCUSSION THREAD

**Discussion by CalvinKoung** [bitcraze/discussions#1226] [Q&A]
Hi, How can I setting the crazyradio to connect to the specific crazyflie ? Because I have the two crazyflie and crazyradio when I connect them simultaneously on different computers. Their connection addresses are the same...

**Comment by gemenerik** [comment]
You can e.g. keep them on the same radio address, but use different channels.

To do that; in the `cfclient` connect to one of the two Crazyflies, go to connect>Configure 2.X, and change the channel. See [the userguide](https://www.bitcraze.io/documentation/repository/crazyflie-clients-python/master/userguides/userguide_client/#firmware-configuration).

**Reply by CalvinKoung** [reply]
Hi, I changed the one crazyflie channel to 100, but I when I scanning I cannot found other channel crazyflie only the channel 100 crazyflie appear. No other crazyflie options are shown. 
![VirtualBox_BitcrazeVM 1_21_03_2024_12_02_14](https://github.com/bitcraze/discussions/assets/118436188/764bf59e-1d46-4da8-b1e5-5a1df9e2a637)

**Reply by ataffanel** [reply]
You should double-check that both Crazyflie are on different channel, spaced by at least 2 but preferably more, and on address 0xe7e7e7e7e7 so that the scan would be able to detect both. Also make sure that both Crazyflie have a channel bellow 100 if you are using a Crazyradio 2.

**Reply by CalvinKoung** [reply]
ok, I got it. Thanks your help
